### DISCUSSION TITLE
What is Helm Operator v2 and how is it accessed?

### DISCUSSION THREAD

**Discussion by saada** [fluxcd/flux2#221] [Q&A] [answered]
I noticed a mention of Helm Operator v2 and was wondering what that was in contrast of Flux v2

**Comment by seaneagan** [comment, marked as answer]
There is a `HelmRelease` v2, but the `Helm Operator` is now rebranded as the [helm-controller](https://toolkit.fluxcd.io/components/helm/controller/). Perhaps we should remove any references to `Helm Operator v2` in the docs to avoid confusion.

**Reply by seaneagan** [reply]
> Perhaps we should remove any references to `Helm Operator v2` in the docs to avoid confusion.

Or clarify them with a helm-controller reference.
